include/seqan3/range/detail/random_access_iterator.hpp

```diff
@@ -76,7 +76,7 @@ class random_access_iterator_base
     //!\brief Iterator stores pointer to underlying container structure.
     typename std::add_pointer_t<range_type> host{nullptr};
     //!\brief Use container's size_type as a position.
-    using position_type =  ranges::v3::size_type_t<range_type>;
+    using position_type =  std::ranges::size_type_t<range_type>;
```

Comment:
This doesn't work. I've included core/metafunction/all.hpp and it gives a whole bunch of errors. @marehr said it might be because of some cyclic dependency checks?